lib/views/pages/profile/posts_info/info_card_comment.dart

```diff
@@ -0,0 +1,46 @@
+import "package:flutter/material.dart";
+
+//info card for the badges
```

Comment:
I think there is a typo here
```suggestion
//info card for the comments
```